index.html

```diff
@@ -1667,24 +1680,30 @@ <h3>Proofs (Signatures)</h3>
         <p>
 At least one proof mechanism, and the details necessary to evaluate that proof,
 MUST be expressed for a <a>credential</a> or <a>presentation</a> to be a
-<a>verifiable credential</a> or <a>verifiable presentation</a>, i.e., to be
-<em>verifiable.</em> This specification
-identifies two classes of proof mechanisms: external proofs and embedded proofs. An
-<dfn>external proof</dfn> is one that wraps an expression of this data model,
-such as a JSON Web Token, which is elaborated upon in Section
-<a href="#json-web-token"></a>. An <dfn>embedded proof</dfn> is a mechanism
-where the proof is included in the data, such as a Linked Data Signature,
-which is elaborated upon in Section <a href="#linked-data-proofs"></a>.
-When embedding a proof, the following <a>property</a> MUST be used:
+<a>verifiable credential</a> or <a>verifiable presentation</a>. That is, to be
```

Comment:
`presentation</a>. That`
-> `presentation</a>; that`

Reply:
Done.